.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,99 @@
+name: "🐞 Bug Report"
```

Comment:
Can you please add to the header of all these for files SPDX tags, to comply to REUSE specification?

Please check other markdown files to get an example.